### PR TITLE
added title attribute to UploadField error

### PR DESCRIPTION
**Pull request by ivoba**
This should fix https://github.com/silverstripe/silverstripe-asset-admin/issues/891

This is a lucky shot PR :) as it seemed to be an easy adjustment.  
I did not build the client to test it and im also not sure if this is correct in react context.
Any advice welcome.

### CODE REVIEW

**Comment by robbieaverill** [comment]
Assuming the `message` inside that element is truncated via CSS, then this is a good change in my books. You'll need to build the dist files though - `yarn build` in the module (requires node 10 for the `1` branch).

File: client/src/components/UploadField/UploadFieldItem.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ class UploadFieldItem extends Component {
 
     if (message !== null) {
       return (
-        <div className="uploadfield-item__error-message">
+        <div className="uploadfield-item__error-message" title="{message}">
```
Comment (robbieaverill) [inline client/src/components/UploadField/UploadFieldItem.js]: Actually, I think it needs to be this:
```suggestion
        <div className="uploadfield-item__error-message" title={message}>
```

**Comment by ivoba** [comment]
@robbieaverill i made a new branch for a new PR but seems i am not able to build.
I ran composer install & yarn install & yarn build (on Node 6) but i get a lot of module not found errors like
`Cannot find module 'components/FormAlert/FormAlert' from 'ThumbnailView-test.js'`

I can close this PR and make a new one as described above. Would you mind building the client?

**Comment by robbieaverill** [comment]
@ivoba you will need to install the dev version of the admin module e.g. 1.x-dev, but yeah I'm happy to build it for you!

**Pull request closed without merging**